202103_bicepkeck/bicepkeck.toml

```diff
@@ -0,0 +1,50 @@
+[experiment]
+
+total_experiment_length_years = 6
+
+
+# Number of splits, 1 generates only full mission
+# 2-7 generates the full mission map and then the requested number
+# of splits scaled accordingly. E.g. 7 generates the full mission
+# map and 7 equal (yearly) maps
+number_of_splits = 1
+
+include_atmosphere = false
+
+include_noise = false
+
+# between 0 and 1, typically 0.2
+# remove default so the code errors if a band is not specified
+# observing_efficiency.default = 0.2
+
+# we can override the observing efficiency for specific sites/bands
+# always specify first site, then telescope, then band
+
+# Copy-pasted from https://docs.google.com/spreadsheets/d/1jR9gSsJ0w1dEO5Jb_URlD3SWYtgFtwBgB3W88p6puo0/edit#gid=1776795091
+# on 4 March 2021
+
+observing_efficiency.Pole.SAT.LFS1  = 0.194656896
+observing_efficiency.Pole.SAT.LFS2  = 0.194656896
+observing_efficiency.Pole.SAT.MFLS1 = 0.194656896
+observing_efficiency.Pole.SAT.MFHS1 = 0.194656896
+observing_efficiency.Pole.SAT.MFLS2 = 0.1914126144
+observing_efficiency.Pole.SAT.MFHS2 = 0.1914126144
+observing_efficiency.Pole.SAT.HFS1  = 0.1359894704
+observing_efficiency.Pole.SAT.HFS2  = 0.0940841664
+
+# we can also specify defaults for sites (notice capitalization) or telescopes
+# observing_efficiency.Pole.SAT.default = 0.2
+# observing_efficiency.Chile.default = 0.2
+
+[telescopes]
+
+    # Telescope names are not used within the software
+
+    [telescopes.SAT]
+
+    # Detector ratio 288/3528
+    BK15_90 =  { LFS=0, MFLS= 0.08163265306, MFHS=0, HFS=0, site="Pole", years=2 }
+    # Detector ratio 512/3528
+    BK15_150 = { LFS=0, MFLS= 0, MFHS=0.14512471655, HFS=0, site="Pole", years=5.6 } 
```

Comment:
yes, Clem said 11.2 years total, so I did 5.6 years x 2, and then scaled for the detector ratio, but I think there is an error,
it should be:

512/3528*2, I missed a factor of 2 it should be 0.2902494331

I think the same error applies to 90:

288/3528*2 = 0.16326530612

and also to 220:

512/8592*2 = 0.11918063314

I must have **replaced** 2 with the detector count ratio instead of multiplying them.
I will wait for Clem's answer on number of telescopes and then fix and rerun.

Reply:
I got this from Clem's old email:

> The posting lists the BK15 number of detector years at 90/150/220 as 288*4,
512*11.2 and 512*2. For 90 that is 2 receivers over 2 seasons. For 220 it's 2
receivers over 1 season. For 150 it's more complex since the 11.2 is an
effective number given that many 150 focal planes severely under performed. I
would suggest the following effective numbers:
2010 1
2011 1
2012 3
2013 2.5
2014 2
2015 1

I am assuming "2 receivers" are in the same telescope.

Reply:
it seems like except for 2012, they always had 1 telescope only. So do you suggest to fix the factors of 2 and rerun?

Reply:
Turns out in 2012, BICEP2 and Keck were observing simultaneously, so the number of telescope years is 7 rather than 6.

Reply:
I am far less concerned of the total detector counts, although it is nice to get the noise levels roughly calculated. 150GHz should definitely use 7 observing years.

Reply:
ok, then what about the detector years? 
I could do 1.6 tubes per 7 years, so I get 11.2 years total